apiary.apib

```diff
@@ -291,7 +291,25 @@ Get a list of departments.
             + (object)
                 + id: `67c576e7-7e6f-465d-b6ab-a864f6e5e95b` (string)
                 + name: `Human Resources` (string)
+                + vat_number: `BE0899623035` (string)
+                + address (Address)
+                + emails (array)
+                    + (object)
+                        + type: `primary` (enum[string])
+                            + Members
+                                + primary
+                                + invoicing
+                        + email: `info@piedpiper.eu` (string)
+                + telephones (array[object])
+                    + type: `phone` (enum[string], required)
```

Comment:
This should be mobile i think since the corresponding database field is gsm

Reply:
only mobile and fax for departments for now

Reply:
In the UI it's called telephone and fax, hence the phone here.

Reply:
😞